index.html

```diff
@@ -5061,6 +5061,143 @@ <h2 id="AudioContextInfo">
+            <code><a href=
+            "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=processorConstructor,
+            P="parameterDescriptors")</code>. If
+            <code><em>parameterDescriptors</em></code> in
+            <code>undefined</code>, skip to the step 9.
+            </li>
+            <li>For each <code><em>descriptor</em></code> in
+            <code><em>parameterDescriptors</em></code> perform the following
+            substeps:
+              <ol>
+                <li>Let <code><em>audioParam</em></code> be a new
+                <a>AudioParam</a> instance with:
+                  <ul>
+                    <li>
+                      <code><em>paramName</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
+                      P="name")</code>.
+                    </li>
+                    <li>
+                      <code><em>defaultValue</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
+                      P="defaultValue")</code>.
+                    </li>
+                    <li>
+                      <code><em>minValue</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
+                      P="minValue")</code>.
+                    </li>
+                    <li>
+                      <code><em>maxValue</em></code> being the result
+                      <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=item,
+                      P="maxValue")</code>.
+                    </li>
+                  </ul>
+                </li>
+                <li>Let <code><em>initialValue</em></code> be the result of
+                <code><a href=
+                "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=<em>options</em>,
+                P=<em>paramName</em>)</code>.
+                  <ul>
+                    <li>If the result of <a href=
+                    "https://tc39.github.io/ecma262/#sec-ecmascript-data-types-and-values">
+                      Type</a><code>(<em>initialValue</em>)</code> is
+                      <code>Number</code>, perform <code><a href=
+                      "https://tc39.github.io/ecma262/#sec-set-o-p-v-throw">Set</a>(O=<em>audioParam</em>,
+                      P="value", V=<em>initialValue</em>)</code>. Otherwise
+                      ignore and go to the next step.
+                    </li>
+                  </ul>
+                </li>
+                <li>Perform <code><a href=
+                "https://tc39.github.io/ecma262/#sec-set-o-p-v-throw">Set</a>(O=<em>baseNodeInstance</em>,
+                P=<em>paramName</em>, V=<em>audioParam</em>, false)</code>.
```

Comment:
Ah OK, now that I understand it, it makes more sense. It's an unusual design, but this is a good way to do it. It would be slightly better if you used [CreateDataProperty](https://tc39.github.io/ecma262/#sec-createdataproperty); that will avoid triggering any setters (e.g. if they create a param named `onmessage` or something similar).